data/json/monstergroups/eggs.json

```diff
@@ -245,11 +245,12 @@
   {
     "name": "GROUP_EGG_BUTTERFLY",
     "type": "monstergroup",
-    "default": "mon_caterpillar",
-    "monsters": [
-      { "monster": "mon_caterpillar_giant", "freq": 25, "cost_multiplier": 1, "starts": "28 days" },
-      { "monster": "mon_caterpillar", "freq": 100, "cost_multiplier": 1 }
-    ]
+    "monsters": [ { "monster": "mon_caterpillar_butterfly" } ]
+  },
```

Comment:
I'm not sure eliminating the giant caterpillar entirely is the best move here. Consider having both giant and regular for both butterfly and moth and growing into their related butterfly and moth size. It may also help to define some larger eggs so this has a  more logical flow